jupyter-extension/jupyterlab_pachyderm/handlers.py

```diff
@@ -570,6 +571,7 @@ def write_config(
         config.add_context(name, context, overwrite=True)
     else:
         config = ConfigFile.new_with_context(name, context)
+        os.makedirs(PACH_CONFIG.parent, exist_ok=True)
```

Comment:
I would take this one step further and create all parent directories as needed. This is how I'd do it:
```
PACH_CONFIG.parent.mkdir(parents=True, exist_ok=True)
```

Reply:
as discussed offline, `os.makedirs` does create the intermediate directories